src/tidyversetopandas/tidyversetopandas.py

```diff
@@ -20,4 +20,5 @@ def arrange(df: pd.DataFrame, *col_name:str):
     >>> df = pd.DataFrame(np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]]), columns=['a', 'b', 'c'])
     >>> df_sorted = tidyversetopandas.arrange(df, 'a', 'c')
```

Comment:
Do not need to have tidyversetopandas for the example
```suggestion
    >>> df_sorted = arrange(df, 'a', 'c')
```

Reply:
Good point. Thanks for checking!